internal/pkg/cli/deploy/static_site.go

```diff
@@ -68,5 +97,26 @@ func (d *staticSiteDeployer) UploadArtifacts() (*UploadArtifactsOutput, error) {
 			return nil, err
 		}
 	}
-	return &UploadArtifactsOutput{}, nil
+	return d.uploadArtifacts()
+}
+
+func (d *staticSiteDeployer) stackConfiguration(in StackRuntimeConfiguration) (cloudformation.StackConfiguration, error) {
```

Comment:
nit: to be consistent w the rest of the deployers
```suggestion
func (d *staticSiteDeployer) stackConfiguration(in *StackRuntimeConfiguration) (cloudformation.StackConfiguration, error) {
```